translations/harbour-determinant-es.ts

```diff
@@ -91,7 +91,7 @@
     <name>MessageInput</name>
     <message>
         <source>Message</source>
-        <translation>Mensaje</translation>
+        <translation>A b c</translation>
```

Comment:
This is probably a mistake

Reply:
A b c is good, it is my own form of spanish.